stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -326,19 +326,41 @@ julia> kron(A, B)
  3+0im  0-3im  4+0im  0-4im
 ```
 """
-function kron(a::AbstractMatrix{T}, b::AbstractMatrix{S}) where {T,S}
-    require_one_based_indexing(a, b)
-    R = Matrix{promote_op(*,T,S)}(undef, size(a,1)*size(b,1), size(a,2)*size(b,2))
+function kron(A::AbstractMatrix{T}, B::AbstractMatrix{S}) where {T,S}
+    R = Matrix{promote_op(*,T,S)}(undef, size(A,1)*size(B,1), size(A,2)*size(B,2))
+    return @inbounds kron!(R, A, B)
+end
+
+"""
+    kron!(C, A, B)
+
+Like [`kron`](@ref), but stores the results in `C`.
+
+!!! tip
```

Comment:
Is this a recognized Documenter thing? I might go with `!!! info` to be safe.

Reply:
Yes, tip will be a green block, but info is good as well.

Reply:
Oh nice, I didn't know that. Either is fine then.